packages/wrangler/src/pages/build.ts

```diff
@@ -121,32 +134,110 @@ export const Handler = async ({
 		}
 	}
 
+	// for now let's restrict `outfile` to `_worker.bundle` if the
+	// experimentalWorkerBundle flag is set. Once we remove the flag
+	// this will no longer be an issue
+	outfile = experimentalWorkerBundle ? "_worker.bundle" : outfile;
 	buildOutputDirectory ??= dirname(outfile);
-	try {
-		await buildFunctions({
-			outfile,
-			outputConfigPath,
-			functionsDirectory: directory,
-			minify,
-			sourcemap,
-			fallbackService,
-			watch,
-			plugin,
-			buildOutputDirectory,
-			nodeCompat,
-			routesOutputPath,
+
+	const workerScriptPath = resolvePath(buildOutputDirectory, "_worker.js");
+	const foundWorkerScript = existsSync(workerScriptPath);
+	let bundle: BundleResult | undefined = undefined;
+
+	// `wrangler pages functions build --experimental-worker-bundle=true`
+	if (
```

Comment:
not sure if necessary. I ran this cmd randomly in all the wrong folders by accident so many times that I got so annoyed and wrote a nice error message. It might come in handy once the flag is removed tho ¯\_(ツ)_/¯

Reply:
Love it!